docfx_project/index.md

```diff
@@ -21,6 +21,7 @@ Puppeteer Sharp is a .NET port of the official [Node.JS Puppeteer API](https://g
 
 # Useful links
 
+* [GitHub-Repo](https://github.com/hardkoded/puppeteer-sharp)
 * Slack channel [#puppeteer-sharp](https://puppeteer.slack.com/join/shared_invite/enQtMzU4MjIyMDA5NTM4LWI0YTE0MjM0NWQzYmE2MTRmNjM1ZTBkN2MxNmJmNTIwNTJjMmFhOWFjMGExMDViYjk2YjU2ZmYzMmE1NmExYzc)

```

Comment:
Do you want to change this link with this?
https://www.hardkoded.com/goto/pptr-slack

I will keep that link updated :)